k8s/baka/templates/statefulset.yaml

```diff
@@ -1,10 +1,11 @@
 apiVersion: apps/v1
-kind: Deployment
+kind: StatefulSet
```

Comment:
![ya](https://media.tenor.com/w1C5KqX-sJoAAAAC/its-as-shrimple-as-that-shrimple.gif)